modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,116 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies implement an additional level of granularity with SLA tiering. Policies control access based on the following access structuring:
```

Comment:
This seems confusing, we are saying that it adds a level of granularity with SLA tiering and then explain the different options which are not reduced to SLA based. Shouldn't we remove the "with SLA tiering"? and say that policies can implement an additional level of granularity and then list the options?

Reply:
Good catch! Done! Thanks!